docs/src/pages/guides/styling.md

```diff
@@ -159,6 +162,15 @@ Now you're ready to write Tailwind! Our recommended approach is to create a `src
 
 As an alternative to `src/styles/global.css`, You may also add Tailwind utilities to individual `pages/*.astro` components in `<style>` tags, but be mindful of duplication! If you end up creating multiple Tailwind-managed stylesheets for your site, make sure you're not sending the same CSS to users over and over again in separate CSS files.
 
+#### v20 Tweaks to the tailwind import
+As of [Version 20](https://github.com/snowpackjs/astro/releases/tag/astro%400.20.0), Astro will no longer bundle, build and process public files. You now need to move your styles into the `src` directory and import them like so, making use of [Astro Resolve](http://localhost:3000/reference/api-reference#astroresolve): 
+```html
+  <link 
+    rel="stylesheet" 
+    href={`${Astro.resolve(../styles/global.css)}`} 
```

Comment:
You are missing quotes around path.
```suggestion
    href={`${Astro.resolve("../styles/global.css")}`}
````

Reply:
+1 the current snippet probably wouldn't load

Reply:
I've added them: de85fce
Oddly they are the break for the html code block haha
So i've removed the string interpolation too. It looks like this: 

![CleanShot 2021-08-30 at 10 24 55](https://user-images.githubusercontent.com/16897508/131270451-9dc59519-4ec7-4834-ac44-cc02ceae6a5d.png)